### PR TITLE
CASMNET-1741:  Adding hmnlb network to the gateway tests

### PR DESCRIPTION
**Pull request by johren-hpe**
## Summary and Scope

Adding HMNLB network to the gateway tests now that the hmn-gateway has authN and authZ.

I added the cray-nls (argo) service to the set of tested services since that was added in 1.3.

I also changed the use-api-gw-override to false so the test is no longer using api-gw-service-nmn.local.   We are starting to move away from that name.

The HMNLB network is not accessible on the UAI so it only ends up trying to get a token and then moving on when it finds that it cannot get the token (as expected).

## Issues and Related PRs

* Resolves CASMNET-1741

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

Ran the gateway tests on both NCN and UAI on surtur.   All UAI tests passed. 

I also ran the tests on an earlier install of surtur where the UAI did not have the HMN and HMNLB blocked.  The test caught this and CASMUSER-3050 was created.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: uai-images/gateway_test_uai/test/gateway-test.py [hunk 1]
```diff
@@ -330,9 +332,13 @@ def get_vs_gateways(vsyaml):
               else:
                   print("SKIP - [" + svcname + "]: " + url + " - gateways not found")
                   continue
-  
+
           if net['gateway'] not in svcgws:
-            svcexp = 404
+            # HMNLB behaves differently than other networks right now (CASMPET-5853)
```
Comment (alanm-hpe) [inline uai-images/gateway_test_uai/test/gateway-test.py:337]: Should we have another ticket linked to CASMPET-5853, so we know when to fix this here?

**Comment by alanm-hpe** [comment]
The `.version` file needs to be bumped as well.

**Comment by johren-hpe** [comment]
.version file updated

**Review by alanm-hpe — APPROVED** [review]

File: uai-images/gateway_test_uai/test/gateway-test.py (hunk 1 above)
Comment (johren-hpe) [reply uai-images/gateway_test_uai/test/gateway-test.py:337]: I was thinking that the gateway tests should be fixed as part of CASMPET-5853.   I added that to the description of the ticket.

**Pull request merged**